uui/components/layout/RadioGroup.tsx

```diff
@@ -1,10 +1,70 @@
-import { withMods } from '@epam/uui-core';
-import { RadioGroup as uuiRadioGroup, RadioGroupProps } from '@epam/uui-components';
-import { RadioInput } from '../inputs/RadioInput';
+import { cx, directionMode, ICanBeReadonly, ICanFocus, IDisableable, IEditable, IHasCX, IHasDirection, IHasForwardedRef, IHasRawProps } from '@epam/uui-core';
+import { RadioInput, RadioInputProps } from '../inputs';
 import css from './RadioGroup.module.scss';
+import * as React from 'react';
 
-export const RadioGroup = withMods<RadioGroupProps<any>>(
-    uuiRadioGroup,
-    () => [css.root],
-    () => ({ RadioInput }),
-);
+export interface RadioGroupItem<TValue> extends IDisableable, Omit<RadioInputProps, 'id' | 'onValueChange' | 'value'> {
+    /** RadioInput label. Can be a string, or React.Element. */
+    name?: string;
+    /** Render callback for checkbox label.
+     * If omitted, 'name' prop value will be rendered.
+     */
+    renderName?: () => React.ReactNode;
+    /** Item ID to put into selection */
+    id: TValue;
+}
+
+export interface RadioGroupProps<TValue>
+    extends IHasCX,
+    IEditable<TValue>,
+    IDisableable,
+    IHasDirection,
+    ICanBeReadonly,
+    IHasRawProps<React.FieldsetHTMLAttributes<HTMLFieldSetElement>>,
+    IHasForwardedRef<HTMLFieldSetElement>,
+    ICanFocus<HTMLFieldSetElement> {
+    /** Overrides the component to render a single radio Input  */
+    RadioInput?: React.ComponentType<RadioInputProps>;
```

Comment:
Now we don't need RadioInput prop, since we are exporting it from uui package

Reply:
fixed